lib/commands/gesture.js

```diff
@@ -24,11 +24,15 @@ helpers.handleTap = async function (gesture) {
     params.y = options.y;
   }
 
-  let endpoint = `/tap/${util.hasValue(options.element) ? options.element : '0'}`;
+  let endpoint = '';
+  if (gesture.action === 'tap') {
+    endpoint = `/${gesture.action}/${util.hasValue(options.element) ? options.element : '0'}`;
+  } else if (gesture.action === 'doubleTap') {
```

Comment:
I know that, given the calling code, there is no way for `gesture.action` to have a value other than `'tap'` or `'doubleTap'`, but the conditional would be more immediately understandable if it had a default case which threw an error.